### DISCUSSION TITLE
How to get other repeaters items

### DISCUSSION THREAD

**Discussion by SalaheddineBoukhoulda** [filamentphp/filament#5837] [Help] [answered]
I have a repeater item containing a Toggle, i want only one toggle in the repeater item list to be active at a time, so if the user checks a toggle i want to use afterStateUpdated and set all the other toggle in the repeater to false, i tried to use the $get Closure but i don't know what to put in the selector since the $get of a repeater is relative to the item itself, i'm new to Filament, any help would be appreciated.

**Comment by joaovdiasb** [comment, marked as answer]
You can access using $get, like $get('../../component_name')

Heres a example:

```php
Toggle::make('is_default')
         ->label('Default')
         ->visible(function($component, callable $get) {
             $itemKey = explode('.', $component->getStatePath())[2];
      
             return collect($get('../../processParticipantTypes'))
                 ->except([$itemKey])
                 ->every(fn($item) => !$item['is_default']);
         })
         ->reactive()
         ->default(false)
```
